ebu_tt_live/scripts/test/test_scripts.py

```diff
@@ -0,0 +1,12 @@
+
+from subprocess import Popen, PIPE
+from unittest import TestCase
+from ebu_tt_live.scripts import ebu_dummy_encoder
+
+
+class TestDummyScript(TestCase):
+
+    def test_simple_run(self):
+        process = Popen('ebu-dummy-encoder', stderr=PIPE, stdout=PIPE)
```

Comment:
This is exactly the reason it is only the dummy encoder that is in this file for the moment. Twisted supports mocking out networking via their twisted.trial package and make it work. I will come to do it when we get a bit more time. #244 stays open for this reason.